src/components/AssignmentCreatorForm.jsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import {t} from 'i18next';
+import classnames from 'classnames';
+import map from 'lodash-es/map';
+import chrono from 'chrono-node';
+
+import PropTypes from 'prop-types';
+import {Field, reduxForm} from 'redux-form/immutable';
```

Comment:
Is `redux-form` a big contributor to bundle size? If it’s at all significant I think we should load it asynchronously, especially since it’s only used by instructors.

The use of `react-coroutine` in [this (sadly unmerged) component](https://github.com/popcodeorg/popcode/blob/19a5f331c50c3c6075d39923821941e70cf39e5c/src/components/InstructionsEditorAsync.jsx) is a great way to accomplish this.

Reply:
For the avoidance of doubt, I _don’t_ like the way that the linked component has both an `<InstructionsEditorAsync>` and a separate `<InstructionsEditor>` component—I think it should just be one component called `<InstructionsEditor>`, which is asynchronous.

Reply:
Got it - how would I determine whether redux-form is a  big contributor to bundle size?